Libplanet.Tests/TestUtils.cs

```diff
@@ -276,6 +254,49 @@ public static byte[] GetRandomBytes(int size)
             return block;
         }
 
+        public static Block<T> AttachStateRootHash<T>(
+            Block<T> block, IStateStore stateStore, IAction blockAction)
+            where T : IAction, new()
+        {
```

Comment:
Why I didn't include this in `TestUtils.MineNext()`, is because there were too many parameters already.